src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Wysiwyg.Edit.cshtml

```diff
@@ -10,11 +10,11 @@
 <style asp-name="trumbowyg" version="2"></style>
 <script asp-src="~/OrchardCore.ContentFields/Scripts/trumbowyg.media.tag.min.js" debug-src="~/OrchardCore.ContentFields/Scripts/trumbowyg.media.tag.js" depends-on="trumbowyg" at="Foot"></script>
 <script asp-name="trumbowyg-shortcodes" at="Foot"></script>
-
+<script asp-src="~/OrchardCore.ContentFields/Scripts/trumbowyg.theme.min.js" debug-src="~/OrchardCore.ContentFields/Scripts/trumbowyg.theme.js" depends-on="trumbowyg, admin-head" at="Foot"></script>
```

Comment:
It's probably better to configure using a configure class, define the dependencies there and just include it here by name.

Reply:
Done